devices/HapticGlove/include/SenseGloveHelper.hpp

```diff
@@ -0,0 +1,281 @@
+/**
+ * @file SenseGloveHelper.hpp
+ * @authors Kourosh Darvish <kourosh.darvish@iit.it>
+ * @copyright 2021 iCub Facility - Istituto Italiano di Tecnologia
+ *            Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ * @date 2021
+ */
+
+#ifndef SENSE_GLOVE_HELPER_HPP
+#define SENSE_GLOVE_HELPER_HPP
+
+#include <Eigen/Dense>
+
+// std
+#include <array>
+#include <iostream>
+#include <memory>
+
+// YARP
+#include <yarp/os/LogStream.h>
+#include <yarp/sig/Vector.h>
+
+// Sense Glove
+#include <DeviceList.h>
+#include <SenseGlove.h>
+
+/**
+ * GloveControlHelper is an helper class for controlling the glove.
+ */
+namespace senseGlove {
+class SenseGloveHelper;
+const std::string LogPrefix = "senseGlove::SenseGloveHelper::";
+const size_t PoseSize = 7;
```

Comment:
@kouroshD Please add any reference that is related to the value `7`

Reply:
I added a comment for this. `// [ position <x, y, z>, quaternion <w, x, y, z> ]`

Reply:
let me know if this is OK